attalos/dataset/vg_prep.py

```diff
@@ -0,0 +1,176 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import json
+import zipfile
+from collections import defaultdict
+from dataset_prep import DatasetPrep, RecordMetadata, SplitType
```

Comment:
It's probably better to refer to this using the fully qualified path?  attalos.dataset.dataset_prep